ui/shared/components/form/FormWizard.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import FormWrapper from './FormWrapper'
+
+class FormWizard extends React.PureComponent {
+
+  static propTypes = {
+    onSubmit: PropTypes.func.isRequired,
+    pages: PropTypes.arrayOf(PropTypes.object),
+  }
+
+  state = { pageIndex: 0, asyncValues: {} }
+
+  navigateNext = () => {
+    this.setState(prevState => ({
+      pageIndex: prevState.pageIndex + 1,
+    }))
```

Comment:
It would be better to support navigating backward as well.

Reply:
Thats a good point

Reply:
added